src/map_field.cpp

```diff
@@ -1498,7 +1498,7 @@ void map::player_in_field( Character &you )
             // Sludge is on the ground, but you are above the ground when boarded on a vehicle
             if( !you.has_trait( trait_GASTROPOD_FOOT ) && ( !you.in_vehicle ) ) {
                 you.add_msg_if_player( m_bad, _( "The sludge is thick and sticky.  You struggle to pull free." ) );
-                you.moves -= cur.get_field_intensity() * 300;
+                you.mod_moves( -cur.get_field_intensity() * 300 );
```

Comment:
this is independant of get_speed()?

Reply:
Yeah, those fields apply a flat move reduction. I don't love it but changing the calculation would be a balance change.